spec/unit/pdk/config_spec.rb

```diff
@@ -82,6 +96,20 @@ def mock_file(path, content)
         it 'returns the value from the bolt configuration' do
           expect(config.user['analytics']['user-id']).to eq(uuid)
         end
+
+        context 'and the bolt configuration is unparsable' do
+          before(:each) do
+            allow(PDK::Config::YAML).to receive(:new).and_call_original
+            allow(PDK::Config::YAML).to receive(:new)
+              .with(file: File.expand_path(bolt_analytics_path))
+              .and_raise(PDK::Config::LoadError)
+          end
+
+          it 'generates a new UUID' do
```

Comment:
Hrmmm...this I'm not sure about.  If we know there's a bolt config but it's unparsable, should we disable our analytics or generate new ID?

Reply:
I think we should treat it as if the file doesn't exist, so the default values will be `disabled => true` and a new UUID.

Reply:
I guess that's the default behaviour so yeah.